src/dragon_eval/evaluation.py

```diff
@@ -24,6 +24,7 @@
 
 from dragon_eval.evalutils.evalutils import ClassificationEvaluation
 from dragon_eval.evalutils.io import FileLoader
+from dragon_eval.evalutils.redaction_score import evaluate_redaction
```

Comment:
Could you move the redaction score script one folder up? "Evalutils" is for compatibility with the old evalutils from Grand Challenge.